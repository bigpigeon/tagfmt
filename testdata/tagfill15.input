//tagfmt -f "json=snake('')|yaml=upper_camel('')|bson=lower_camel('')"

package main

type OrderDetail struct {
    ID       string   `json:"Id"`
    UserName string   `json:"User"`
}
