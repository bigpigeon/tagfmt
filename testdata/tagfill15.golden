//tagfmt -f "json=snake('')|yaml=upper_camel('')|bson=lower_camel('')"

package main

type OrderDetail struct {
	ID       string `json:"" bson:"" yaml:""`
	UserName string `json:"" bson:"" yaml:""`
}
